HPXMLtoOpenStudio/resources/hpxml_defaults.rb

```diff
@@ -3299,6 +3304,7 @@ def self.get_nbeds_adjusted_for_operational_calculation(hpxml_bldg)
   def self.get_default_flue_or_chimney_in_conditioned_space(hpxml_bldg)
     # Check for atmospheric heating system in conditioned space
     hpxml_bldg.heating_systems.each do |heating_system|
+      next if heating_system.heating_system_fuel == HPXML::FuelTypeElectricity
```

Comment:
This (and the same line below) are additional changes beyond the default location, right? Before, we could have electric heating or water heating system with a flue; wow we cannot?

Reply:
No, this shouldn't change anything. The checks on efficiency (AFUE/EF) would already filter out electric systems, I"m just making the code more explicit/understandable now.

Reply:
I see. 100% efficiency - no flue, e.g.